modules/assim.batch/R/pda.utils.R

```diff
@@ -256,12 +256,23 @@ pda.load.priors <- function(settings, con, path.flag = TRUE) {
     for (i in seq_along(settings$pfts)) {
       
       files <- dbfile.check("Posterior", settings$pfts[[i]]$posteriorid, con, settings$host$name)
+      
       pid <- grep("post.distns.*Rdata", files$file_name)  ## is there a posterior file?
       if (length(pid) == 0) {
         pid <- grep("prior.distns.Rdata", files$file_name)  ## is there a prior file?
       }
       if (length(pid) > 0) {
         prior.paths[[i]] <- file.path(files$file_path[pid], files$file_name[pid])
+      } else {
```

Comment:
```
files <- dbfile.check("Posterior", settings$pfts[[i]]$posteriorid, con, settings$host$name)
```
this line collects the file information for `settings$host$name`. Because of this, when I do a PDA run in test-pecan, I can't load posterior/prior objects that are created in pecan2, even though they're under /fs/data2 or /fs/data3 which are accessible from both test-pecan and pecan2. 

I think you're having a similar problem, your posterior/prior id inserted to the DB by another machine, and although they're probably accessible to your current workflow, code can't load it?

Unless you can think of a solution right away, I can look at how we try to load posterior/prior objects here